### ISSUE TITLE
[Header control Web- All Microsoft]: 'Software', 'PCs and Devices'.... 'Other' label under 'All Microsoft' button is not associated with their respective list items.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7314]
https://microsoft.visualstudio.com/OS/_workitems/edit/37785944

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.
